automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,25 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$filePath32  = "$toolsPath\Git-2.11.0-32-bit.exe"
+$filePath64  = "$toolsPath\Git-2.11.0-64-bit.exe"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$pp = Get-PackageParameters
+Set-InstallerRegistrySettings $pp
+
+Stop-GitSSHAgent
+
+$packageArgs = @{
+    PackageName    = 'git.install'
+    FileType       = 'exe'
+    SoftwareName   = 'Git version *'
+    File           = $installFile
+    SilentArgs     = "/VERYSILENT", "/SUPPRESSMSGBOXES", "/NORESTART", "/NOCANCEL", "/SP-", "/LOG", (Get-InstallComponents $pp)
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
+rm -Force $filePath32, $filePath64 -ea 0
```

Comment:
> Do not generally use non read-only aliases. Aliases such as cat, rm, ps, ls, kill etc are OK as their behavior is well known on multiple OS.

Consider your audience. This is a Windows package manager. You and I are familiar with these commands, but that doesn't mean that others are. So I would disagree on these being well-known for the Windows community.

However I do agree that  `%`, `?`, `select` are widely used in scripts and in frameworks.